word_counter.rb

```diff
@@ -0,0 +1,38 @@
+def word_counter(filepath)
```

Comment:
`word_counter` would be a good name for a class.  The best names for methods are either nouns representing the result, or verbs representing the action taken.